core/src/mindustry/entities/abilities/ShieldArcAbility.java

```diff
@@ -105,6 +116,14 @@ public void init(UnitType type){
 
     @Override
     public void draw(Unit unit){
+        //Display overload progress when reloading the shield
+        Vec2 t = Tmp.v1.set(x, y).rotate(unit.rotation - 90f).add(unit);
+        if(data < 0) {
```

Comment:
This shouldn't be here - other types of shields don't have any kind of progress indicator, it's out of place.